Source/astcenccli_image_external.cpp

```diff
@@ -66,9 +66,9 @@ static void astcenc_runtime_assert(bool condition)
     }
 }
 
-#include "stb_image.h"
-#include "stb_image_write.h"
-#include "tinyexr.h"
+#include "third_party/stb_image.h"
```

Comment:
Please can you change the name of the directory from `third_party` to `ThirdParty` so this follows the existing convention in the project.

Reply:
fixed